common/source/docs/common-gps-for-alt.rst

```diff
@@ -0,0 +1,26 @@
+.. _common-gps-for-alt:
+
+==============================
+Using GPS for Altitude Sensing
+==============================
+
+Until version 4.1, ArduPilot required a barometer altitude sensor. Now, it is possible to boot an autopilot without barometer present, and use GPS (outdoor use) or other altitude sensor (rangefinder, range beacon) for indoor use. 
```

Comment:
I suspect some will be confused by this line and think that we only supported alt from baro in versions before 4.1.. maybe we should just remove the mention of the version and the past situation.

"This page explains how to setup GPS for use as an altitude source."

Reply:
modified it a bit